integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/Compactor.scala

```diff
@@ -106,6 +108,10 @@ object Compactor {
     }
 
     if (finalMergeStatus) {
+      val mergedLoadNumber = mergedLoadName
```

Comment:
Can you add this logic in `LoadMetadataDetails.getLoadNumber` so that it can be used here?

Reply:
I have extracted out a method